maze_transformer/evaluation/plot_maze.py

```diff
@@ -131,6 +137,23 @@ def __init__(self, maze: LatticeMaze) -> None:
         self.target_token_coord: Coord = None
         self.preceding_tokens_coords: CoordArray = None
 
+        if isinstance(maze, TargetedLatticeMaze):
+            self.add_true_path(SolvedMaze.from_targeted_lattice_maze(maze).solution)
+
+        if isinstance(maze, SolvedMaze):
+            self.add_true_path(maze.solution)
+
+    @property
+    def solved_maze(self) -> SolvedMaze:
+        if self.true_path is None:
+            raise ValueError(
+                "Cannot return SolvedMaze object without true path. Add true path with add_true_path method."
```

Comment:
Nitpick: Elsewhere in the repo we call this the "solution", not the "true path". Would be good to have consistent naming.

Reply:
Out of scope for now, this comes from an earlier PR. should change this later.